src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -1285,15 +1034,15 @@ void AppHost::_WindowSizeChanged(const winrt::Windows::Foundation::IInspectable&
     _resizeWindow(_window->GetHandle(), { args.Width(), args.Height() });
 }
 
-void AppHost::_SummonWindowRequested(const winrt::Windows::Foundation::IInspectable& sender,
+void AppHost::_SummonWindowRequested(const winrt::Windows::Foundation::IInspectable&,
                                      const winrt::Windows::Foundation::IInspectable&)
 {
-    const Remoting::SummonWindowBehavior summonArgs{};
+    const winrt::TerminalApp::SummonWindowBehavior summonArgs{};
     summonArgs.MoveToCurrentDesktop(false);
     summonArgs.DropdownDuration(0);
-    summonArgs.ToMonitor(Remoting::MonitorBehavior::InPlace);
+    summonArgs.ToMonitor(winrt::TerminalApp::MonitorBehavior::InPlace);
     summonArgs.ToggleVisibility(false); // Do not toggle, just make visible.
-    _HandleSummon(sender, summonArgs);
+    HandleSummon(summonArgs);

```

Comment:
```suggestion
    HandleSummon(std::move(summonArgs));
```

like in the other place?